chromium_src/chrome/browser/ui/views/side_panel/side_panel_web_ui_view.h

```diff
@@ -0,0 +1,17 @@
+/* Copyright (c) 2023 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_CHROMIUM_SRC_CHROME_BROWSER_UI_VIEWS_SIDE_PANEL_SIDE_PANEL_WEB_UI_VIEW_H_
+#define BRAVE_CHROMIUM_SRC_CHROME_BROWSER_UI_VIEWS_SIDE_PANEL_SIDE_PANEL_WEB_UI_VIEW_H_
+
+#define BRAVE_SIDE_PANEL_WEB_UI_VIEW_H \
+ protected:                            \
+  void AddedToWidget() override;
+
+#include "src/chrome/browser/ui/views/side_panel/side_panel_web_ui_view.h"  // IWYU pragma: export
+
+#undef BRAVE_SIDE_PANEL_WEB_UI_VIEW_H
```

Comment:
With this, we could avoid `side_panel_web_ui_view.h` patch file.
```
#include "ui/views/controls/webview/webview.h"

#define ViewHierarchyChanged \
  AddedToWidget() override;  \
  void ViewHierarchyChanged

#include "src/chrome/browser/ui/views/side_panel/side_panel_web_ui_view.h"  // IWYU pragma: export

#undef ViewHierarchyChanged
```

Reply:
Ah, I missed that we could include a dependency first to avoid interfering with parent class identifiers. Nice!